drivers/vmwarevsphere/cloudinit.go

```diff
@@ -293,3 +308,11 @@ func (d *Driver) addSSHUserToYaml(sshkey string) (string, error) {
 	}
 	return string(yaml), nil
 }
+
+func binaryPathLookup(name string) (string, error) {
+	path, err := exec.LookPath(name)
+	if err != nil {
+		return "", fmt.Errorf("binaryPathLookup: error returned when trying to find [%s] executable: [%s]", name, err.Error())
```

Comment:
There is an obscure situation where this can panic. Essentially, `err != nil` doesn't guarantee that `err.Error()` won't panic. Adjusting this will avoid that:
```suggestion
		return "", fmt.Errorf("binaryPathLookup: error returned when trying to find [%s] executable: [%v]", name, err)
```